app/controllers/concerns/manifest_params.rb

```diff
@@ -1,5 +1,12 @@
 module ManifestParams
+  def deep_reject!(hash, &block)
```

Comment:
noted. don't think we should get rid of the method, just want to see it under test so we / I don't break its functionality in the future